src/lib.rs

```diff
@@ -47,46 +41,27 @@ where
 {
     /// Invokes a specific action on the actor.  Returns a future that completes when the actor has
     /// performed the action
-    pub fn invoke(&self, action: A) -> ActFuture<R, E> {
+    pub async fn invoke(&self, action: A) -> ActResult<R, E> {
```

Comment:
This should be callable in a "fire-and-forget" fashion, i.e. it's not `async` but instead returns a `Future` that is optional to poll depending on the client requirements.